app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -57,7 +57,9 @@
       .form-group
         .col-md-12{"ng-if" => "#{ng_model}.ems_controller == 'ems_container' || " +  |
                               "#{ng_model}.ems_controller == 'ems_physical_infra' || " + |
-                              "#{ng_model}.ems_controller == 'ems_network'"}         |
+                              "#{ng_model}.ems_controller == 'ems_block_storage' || " + |
```

Comment:
Also unnecessary...

Reply:
We did this work a while back, I'll go over it and remove the deprecated code.

Reply:
removed